frontend/src/common/ReviewWithComment.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button, Dialog, DialogContent, DialogContentText, DialogTitle, Stack, TextField } from '@mui/material'
+import { useState } from 'react'
+
+type ReviewWithCommentProps = {
+  open: boolean
+  onClose: () => void
+  onSubmit: (kind: string, reviewComment: string) => void
+  title: string
+  description: string
+}
+
+enum ResponseTypes {
+  APPROVE = 'approve',
```

Comment:
In most other enums (I realise we're not great at this) we usually use title case for the enum name. e.g. `Approve = 'approve'`